test/e2e/scenario/agents.scenario.ts

```diff
@@ -100,7 +100,7 @@ describe('rum', () => {
 
     // Edge 18 seems to have valid timings even on cross origin requests ¯\_ツ_/¯ It doesn't matter
     // too much.
-    if (browser.capabilities.browserName === 'MicrosoftEdge' && browser.capabilities.browserVersion === '18') {
+    if (browser.capabilities.browserName === 'MicrosoftEdge') {
```

Comment:
I'd be in favor of having a condition taking the chromium-based Edge versions into account. Having a fixed version works: if the "browserVersion" isn't avaibable anymore, in any other capabilities property?
Else, does chromium-based Edge have a different "browserName"?

Reply:
I removed the check because version is handled differently by bs.
My take on that is that when we will upgrade edge version, this test will fail and we could remove this check instead of forgetting it.

Reply:
IMO, we'll have to support both Chromium and non-Chromium Edge versions before remoing the non-Chromium Edge support, so we'll need to adjust the check anyway. But I'm fine to wait until then :)